noxfile.py

```diff
@@ -0,0 +1,132 @@
+import nox
+import os
+
+# Use system installed Python packages
+PYT_PATH='/opt/conda/lib/python3.8/site-packages'
+
+# Root directory for torch_tensorrt. Set according to docker container by default
+TOP_DIR='/opt/pytorch/torch_tensorrt'
```

Comment:
How would this work for developers?

Reply:
This is targeted for docker container. For developers we can make this configurable using os.environ.get('TOP_DIR')

Reply:
I think ideally we make the default for developers since it is easy to add env variables to a script or something running the tests

Reply:
Done.